application/views/index.php

```diff
@@ -1,3 +1,3 @@
 <a href="<?=site_url('registration')?>">
-	<img src="<?=base_url()?>/assets/images/content.jpg">
+	<img src="<?=base_url()?>/assets/images/content.jpg" width="1100">
```

Comment:
Actually, I do not think we should use absolute width, cause it is not friendly to different screen size.